### ISSUE TITLE
<a name"6.3.6"> in release notes

### ISSUE THREAD

**Issue by ethanneff** [semantic-release/semantic-release#363]
Why does semantic release do this? It is even in your own repo

<img width="751" alt="screen shot 2017-02-23 at 7 42 45 am" src="https://cloud.githubusercontent.com/assets/2933593/23266139/b2121b7a-f99b-11e6-96b0-f9be5b96e473.png">

**Comment by honzajavorek** [comment]
It was super annoying to go down the dependency chain and track this down, but it looks like the package which generates the actual text is `conventional-changelog/conventional-changelog-writer` and the template is [here](https://github.com/conventional-changelog/conventional-changelog-writer/blob/master/templates/header.hbs#L1). But in the template it's correct. It looks like current template is correct and Sem Rel needs to update dependencies (https://github.com/conventional-changelog/conventional-changelog-writer/commit/9942863d93d609899c23481798fd9a649138f4d6). Not sure how though, it's like 5th dependency down the chain 😐

**Comment by gund** [comment]
It is still written in the release notes... Does anyone know when this can be fixed?

**Comment by enriquecaballero** [comment]
+1

**Comment by coderbyheart** [comment]
See https://github.com/semantic-release/release-notes-generator/issues/4

**Comment by dfbaskin** [comment]
I submitted a PR to the `release-notes-generator` repo.  All it does it update the version of `conventional-changelog`.

https://github.com/semantic-release/release-notes-generator/pull/5

The build succeeded and the tests passed.  If that can get merged, then I can submit a PR here.

**Comment by buehler** [comment]
@dfbaskin Awesome.. now some guy should merge it :)

**Comment by dfbaskin** [comment]
@bahmutov That works nicely.  So is `release-notes-generator` deprecated now?

**Comment by buehler** [comment]
@dfbaskin @bahmutov 
I tried it :-) It works quite well, but the generation of the release notes are a little bit off:
![247b6_8j](https://user-images.githubusercontent.com/1067213/28151692-74434dd6-679c-11e7-8a76-5a957893e2ba.png)

The "old" generator does group the notes by title and scope, which is a pretty nice feature.

**Comment by uglow** [comment]
@bahmutov Why was the grouping behaviour (by type & scope) removed?

**Comment by bahmutov** [comment]
Hi! for changelog, I am just calling conventional changelog and have no idea why this version just posts the commits list

See https://github.com/bahmutov/github-post-release/blob/8e7cb4adcb12fd6e19d7c3440fa51123517f7476/src/index.js#L136
```js
  const generateChangeLog = () => {
    debug('generate changelog')
    return streamToPromise(
      changelog({
        version: pkg.version,
        repository: parsedRepo,
        fail: false
      })
    ).then(buffer => buffer.toString())
  }
```

I do want to fix this, so there is an issue bahmutov/github-post-release#9

**Comment by bahmutov** [comment]
@dfbaskin i have no idea, this project is really on very low maintenance right now, seems to me

**Comment by bahmutov** [comment]
ok, updated the changlog and factored it into https://github.com/bahmutov/simple-changelog and examples of release notes can be found in https://github.com/bahmutov/github-post-release/releases

Note, I am using https://github.com/bahmutov/simple-commit-message convention which is simpler than conventional one

**Comment by gr2m** [comment]
Hey folks, I’m looking into this now, I’ll help maintain the project moving forward.

**Comment by gr2m** [comment]
I’ll keep you all posted in this issue moving forward: https://github.com/semantic-release/semantic-release/issues/411 Thanks for your patience and if you could help triaging open issues / PRs and mention me in the ones you think are most urgent, I’d appreciate it <3

**Comment by gr2m** [comment]
I’ve released `semantic-release@7.0.0` with the next release. You can install it with `npm install --save-dev semantic-release@next` or directly set the version in your projects’ `package.json` files.

This should resolve the issue. Please give it a try and let me know if it works as expected, once a few folks tested it we can set the `semantic-release@7.0.0` to `@latest`

**Comment by vanduynslagerp** [comment]
Fixed in last `semantic-release` version
